KlayGE/Core/Include/KlayGE/Input.hpp

```diff
@@ -53,6 +57,9 @@
 #elif defined(KLAYGE_COMPILER_GCC)
 #pragma GCC diagnostic pop
 #endif
+#if defined(KLAYGE_COMPILER_CLANG)

```

Comment:
Those 3 lines should be moved to the line after #include <boost/container/flat_map.hpp>